test/TimeoutTests/Python/proxies.json

```diff
@@ -0,0 +1,4 @@
+{
+  "$schema": "http://json.schemastore.org/proxies",
+  "proxies": {}
+}
```

Comment:
Are you using the proxies feature for something?

Reply:
Not sure why this is there or if it is needed. I just did a copy/paste of the durablePy project inside SmokeTests, which seemed convenient.

Reply:
This is automatically added by the scaffolding in VSCode. I think it can be safely removed

Reply:
Let's remove this. It might actually break something down the road if we leave it here.